README.md

```diff
@@ -0,0 +1,168 @@
+<!--
+*** Thanks for checking out the DTonomy github page.
+*** Please visit us at https://www.dtonomy.com for more information. 
+-->
+[![Apache-2.0 License][license-shield]][license-url]
+[![Website][website-shield]][website-url]
+[![Twitter][twitter-shield]][twitter-url]
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation-example">Installation Example</a></li>
+      </ul>
+    </li>
+    <li><a href="#list-of-nodes">List of Nodes</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+This project contains third-party nodes which are created by DTonomy to be used in the security automation process. 
```

Comment:
This project contains third-party nodes created by DTonomy to be used for automating security analysis and response